awx/main/tests/functional/api/test_credential.py

```diff
@@ -60,6 +60,36 @@ def test_credential_validation_error_with_bad_user(post, admin, credentialtype_s
     assert response.data['user'][0] == 'Incorrect type. Expected pk value, received str.'
 
 
+@pytest.mark.django_db
+def test_credential_validation_error_with_no_owner_field(post, admin):
+    params = {
+        'credential_type': 1,
+        'inputs': {'username': 'someusername'},
+        'name': 'Some name',
+    }
+    response = post(reverse('api:credential_list'), params, admin)
+    assert response.status_code == 400
+    assert response.data['detail'] == "Missing 'user', 'team', or 'organization'."
```

Comment:
Not sure how to access the detail information. Any help is appreciated.

I am trying to get my local dev env working so I can test locally.

Reply:
it might be in some other key like "msg" but it's hard to know without running and just printing out `response.data`